run_inference.py

```diff
@@ -110,7 +110,7 @@ def main():
                 imwrite(filename + '.png', to_save)
             if args.output_value in ['raw', 'both']:
                 # Make the flow map a HxWx2 array as in .flo files
-                to_save = flow_output.cpu().numpy().transpose(1,2,0)
+                to_save = (args.div_flow.flow_output).cpu().numpy().transpose(1,2,0)
```

Comment:
you made a typo, it should be args.div_flow * flow_output

Reply:
embarrassing on my part ... sorry fixed